packages/roosterjs-editor-plugins/lib/BlockQuote/BlockQuote.ts

```diff
@@ -0,0 +1,75 @@
+import { getBlockQuoteElement, getNodeAtCursor } from 'roosterjs-editor-api';
```

Comment:
I don't think we need this complex design. When type in quoted text, just let it be, no need to do special handling. And it is too heavy to handle each key event for quote.

Reply:
Moved the plugin to TabIndent.